CMakeLists.txt

```diff
@@ -1,157 +0,0 @@
-# options
-option(WITH_TESTS "Choose if CUnit tests should be built" TRUE)
-
-
-#------------------------------------------------------------------------------
-# check for headers
-
-include (CheckIncludeFiles)
-
-check_include_files (stdint.h HAVE_STDINT_H)
-check_include_files (getopt.h HAVE_GETOPT_H)
-
-
-#------------------------------------------------------------------------------
-# PostgreSQL server paths come from pg_config
-#
-
-find_program(PG_CONFIG pg_config)
-if(NOT PG_CONFIG)
-  message(FATAL ERROR "Unable to find 'pg_config'")
-endif(NOT PG_CONFIG)
-
-exec_program(${PG_CONFIG} ARGS --version OUTPUT_VARIABLE PGCONFIG_OUTPUT)
-
-string(REGEX REPLACE 
-    "^PostgreSQL[\t ]+([0-9]+)\\.([0-9]+)\\.([0-9]+)" "\\1.\\2.\\3"
-    PGSQL_VERSION "${PGCONFIG_OUTPUT}")
-
-unset(PGCONFIG_OUTPUT)
-
-if(PGSQL_VERSION)
-    string(REGEX REPLACE
-        "([0-9]+)\\.([0-9]+)\\.([0-9]+)" "\\1"
-        PGSQL_VERSION_MAJOR "${PGSQL_VERSION}")
-    string(REGEX REPLACE
-        "([0-9]+)\\.([0-9]+)\\.([0-9]+)" "\\2"
-        PGSQL_VERSION_MINOR "${PGSQL_VERSION}")
-
-    set(PGSQL_NUMERIC_VERSION ${PGSQL_VERSION_MAJOR}${PGSQL_VERSION_MINOR})
-endif()
-
-exec_program(${PG_CONFIG} ARGS --includedir
-    OUTPUT_VARIABLE PGSQL_INCLUDEDIR)
-exec_program(${PG_CONFIG} ARGS --includedir-server
-    OUTPUT_VARIABLE PGSQL_INCLUDEDIR_SERVER)
-    
-exec_program(${PG_CONFIG} ARGS --pkglibdir OUTPUT_VARIABLE PGSQL_PKGLIBDIR)
-exec_program(${PG_CONFIG} ARGS --sharedir OUTPUT_VARIABLE PGSQL_SHAREDIR)
-exec_program(${PG_CONFIG} ARGS --bindir OUTPUT_VARIABLE PGSQL_BINDIR)
-exec_program(${PG_CONFIG} ARGS --cflags OUTPUT_VARIABLE PGSQL_CFLAGS)
-exec_program(${PG_CONFIG} ARGS --ldflags OUTPUT_VARIABLE PGSQL_LDFLAGS)
-exec_program(${PG_CONFIG} ARGS --libs OUTPUT_VARIABLE PGSQL_LIBS)
-
-#------------------------------------------------------------------------------
-# libxml2
-
-find_package (LibXml2 REQUIRED)
-
-
-#------------------------------------------------------------------------------
-# zlib
-
-find_package (ZLIB REQUIRED)
-
-
-#------------------------------------------------------------------------------
-# cunit, ght and lazperf
-
-if (WITH_TESTS)
-    find_package (CUnit)
-endif (WITH_TESTS)
-
-find_package (LibGHT)
-
-if (LIBGHT_FOUND)
-  set (HAVE_LIBGHT 1)
-  include_directories (${LIBGHT_INCLUDE_DIR})
-endif (LIBGHT_FOUND)
-
-find_package (LazPerf)
-
-if (LAZPERF_FOUND)
-  set (HAVE_LAZPERF 1)
-  include_directories (${LAZPERF_INCLUDE_DIR})
-endif (LAZPERF_FOUND)
-
-#------------------------------------------------------------------------------
-# define targets for "check" and "installcheck"
-
-if (WITH_TESTS)
-    add_custom_target(check COMMAND ${PROJECT_BINARY_DIR}/lib/cunit/cu_tester)
-
-    find_program(PG_REGRESS pg_regress ${PGSQL_PKGLIBDIR}/pgxs/src/test/regress/ REQUIRED)
-    add_custom_target(installcheck COMMAND ${PG_REGRESS} --inputdir=${PROJECT_SOURCE_DIR}/pgsql --outputdir=${PROJECT_SOURCE_DIR}/pgsql pointcloud pointcloud_columns schema)
```

Comment:
Do we have something similar with autotools?

Reply:
https://github.com/pgpointcloud/pointcloud/blob/master/Makefile#L10-L11

Reply:
OK, good to know